api/v1beta1/awscluster_webhook.go

```diff
@@ -114,6 +114,12 @@ func (r *AWSCluster) ValidateUpdate(old runtime.Object) error {
 					r.Spec.ControlPlaneLoadBalancer.Name, "field is immutable"),
 			)
 		}
+		if oldC.Spec.ControlPlaneLoadBalancer.HealthCheckProtocol.String() != ClassicELBProtocolSSL.String() && newLoadBalancer.HealthCheckProtocol != nil {
+			allErrs = append(allErrs,
+				field.Invalid(field.NewPath("spec", "controlPlaneLoadBalancer", "healthCheckProtocol"),
+					r.Spec.ControlPlaneLoadBalancer.HealthCheckProtocol, "field is immutable once set"),
```

Comment:
Use `newLoadBalancer` instead of `r.Spec.ControlPlaneLoadBalancer` --> same logic as above

Reply:
Done